src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let currentState = { ...state };
+  const result = [];
+
+  for (const action of actions) {
+    let newState = { ...currentState };
```

Comment:
redundant
```suggestion
```

Reply:
redundant variable?

Reply:
yes, since I've written below, that you can just use it at `push`
https://github.com/mate-academy/js_stateful-clones/pull/3101/files#r1440814282

Reply:
exactly!!)) that's right, thanks for the hint, I didn't notice it right away, it simplified the code)))